esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -217,6 +219,15 @@ bool CLoggingManager::updateLog(IEspContext* espContext, IEspUpdateLogRequestWra
     return bRet;
 }
 
+bool checkSkipThreadQueue(IPropertyTree *scriptValues, const char *controlName)
+{
+    if (!scriptValues || !controlName || !*controlName)
+        return false;
+    VStringBuffer controlAttr("@%s", controlName);
+    const char *controlValue = scriptValues->queryProp(controlAttr);
+    return (controlValue && strieq(controlValue, "skip"));
```

Comment:
So a script would direct the manager to skip logging for a particular agent by adding a scriptValues attribute like thread-queue-<service>-<agent>="skip"

Reply:
@asselitx I added set-log-option in my previous PR.. it's already in 7.10.x